code/Model/SiteTree.php

```diff
@@ -339,7 +340,16 @@ class SiteTree extends DataObject implements PermissionProvider, i18nEntityProvi
      * @config
      * @var string
      */
-    private static $meta_generator = 'SilverStripe - https://www.silverstripe.org';
+    private static $meta_generator = 'Silverstripe CMS';
+
+    /**
+     * Whether to display the version portion of the meta generator tag
+     * Set to false if it's viewed as a concern.
+     *
+     * @config
+     * @var bool
+     */
+    private static $show_meta_generator_version = true;
```

Comment:
Why not just keep tho old behavior and not output the meta tag if `meta_generator' is blank?

Reply:
That behavior is still there, if you config `private static $meta_generator` to a blank string then there won't be a meta tag for generator

i.e.
```php

    public function MetaComponents()
...
        $generator = $this->getGenerator();
        if ($generator) {
            $tags['generator'] = [
...
    }

    private function getGenerator(): string
    {
        $generator = trim(Config::inst()->get(self::class, 'meta_generator'));
        if ($generator === '') {
            return '';
        }
        if (self::config()->get('show_meta_generator_version')) {
        ...
    }
```

Reply:
I just don't understand what kind of user would say "I don't want the version number there, but I'm fine advertising the site uses Silverstripe CMS".

My gut feeling is the vast majority of users who are bothered by the version number will want to remove the entire meta tag.

Reply:
Probably very few users.  However `private static $meta_generator = 'Silverstripe CMS';` is currently a configurable string, so we do support changing this to something else e.g. 'CMS from Mars'.  

If people want to report this as a different CMS for whatever reason, then I can see why they wouldn't want to include the version number

I'm not hugely attached to having these two things separately configurable, though I also don't see it as much extra hassle